interview-questions.md

```diff
@@ -4,56 +4,90 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: it is a way for us to call keys in state without having to type out the full this.state. syntax everytime we nee need something.
```

Comment:
Just about! I think it's important to note its used not only in our React applications but as a way to help simplify our objects for later use in all areas of Javascript where an object could be used.